아이콘 추가,  selectChips 계속 진행중

opp1350 opened the pull request:
다른분들께 공유/질문하려고 풀 리퀘스트했습니다 ㅠㅠ 머지를 위한게 아니니 나~중에 부탁드려요.. 

selectChips에 데이터를 바인딩 할 수 있게 만들고 싶은데 너무 어려워서 헤매고 있습니다..

그니까 Home.tsx의 60번째 줄처럼 
Chips라는 컴포넌트를 import하여 각 페이지에 맞는 데이터를 바인딩 할 수 있었으면 좋겠는데,
방식자체가 잘못 된 건지 갈피를 못잡겠네요ㅋㅋㅋㅋㅋㅋㅋㅋ 
뭔가 좀 더 좋은 방법이나... 기타 의견을 공유해주신다면.. 어떻게든 제가 붙잡고 해결해 보겠습니다.

아이콘들을 svg로 만들어놓고 싶은데 찾아보니까.. 
조금 복잡하고 자꾸 오류가 나서 일단 png 파일로 추가해 놓았습니다.

opp1350 committed:
아이콘  ts 추가, 아이콘 파일 add
selectChips 를 데이터 바인딩하는 방식으로 변경하고 싶은데 잘 안되네요.
아무튼 진행중. 오류날 코드는 없습니다.

opp1350 committed:
하단 네비게이션 기본 커스텀 ing

sungjin96 commented on front/components/SelectChips.tsx:
```diff
@@ -10,21 +12,21 @@ interface CheckedProps {
 }
 
-function Chips() {
+const Chips = ({states, setStates}) => {
```
function: 으로 핬다가 const로 바꾼 이유가 궁굼하군요

opp1350 replied:
사실 특별한 이유는 없었고 이것저것해보다가 변경된 것 같습니다... 정확한 차이점을 아직 잘 모르는데 고치면서 더 찾아볼게요

sungjin96 commented on front/components/SelectChips.tsx:
```diff
@@ -34,17 +36,31 @@ function Chips() {
-		<Chips onPress={handleToggle} checkedTest={checked}>
-			<ChipsText checkedTest={checked}>여기에 값이 들어감</ChipsText>
+	// onPress={handleToggle}
+	const statesList = states.map((state) => 
```
음.. 일단 사람들마다 다르지만 저는
Chips 라는 컴포넌트를 따로 만들고 ChipsGroup컴포넌트를 또 따로 만들꺼 같습니다.
그 다음 Chips라는 컴포넌트엔 true or false인 value라는 이름으로 프롭스를 넘겨주고 setState를 넘기는게 아니라 그 기능을 해주는 함수를 만들어서 함수를 넘길꺼 같습니다

여기서 함수는 공통으로 사용할수 있게 설계? 해서 만들꺼 같습니다

opp1350 replied:
네 어떤 말씀인지 알겠습니다! (아직 덜 고쳐서 커밋은 안했습니다)

opp1350 committed:
페이지 추가 및 하단 네비게이션에 반영 (+아이콘)

opp1350 commented:
여럿이 동시에 작업하다보니.. 아이콘 이미지의 경로라든지 기타 소소한 부분이 통일/정리되지 않은 상태입니다. 작업하면서 계속 정리해 나갈 예정입니다.

opp1350 committed:
최신 프론트 소스 일괄 업데이트
- 북마크 화면 진행중
- 유저 화면 진행중
- 북마크, 유저화면에 필요한 tab을 인스톨함
- 자잘한 에러 나던 것을 수정 (text가 view안에 있었다든가, key가 없다든가 등등)
- 복잡해지는 컴포넌트 파일 정리 (진행중)

sungjin96 commented on front/components/Card.tsx:
```diff
@@ -0,0 +1,207 @@
+  )
+}
+
+const CardWrap = styled.View`
```
styled component 가 몇개 안되면 그냥 지금처럼 한 파일에 해도 괜찮지만 지금 처럼 많다면 파일 하나 빼서 관리해주는 편이 더 좋을꺼에요!

sungjin96 reviewed (commented):
대부분 icon을 constants 에 선언해놓고 잘 사용하고 있으신거 같은데 몇개는 다르게 이미지를 받아 오시더라구요!! 다 통일 하는 편이 좋을꺼 같아요
